mllib/src/main/scala/org/apache/spark/mllib/evaluation/BinaryClassificationMetrics.scala

```diff
@@ -51,6 +51,7 @@ class BinaryClassificationMetrics(
 
   /**
    * Defaults `numBins` to 0.
+   * @since 1.3.0
```

Comment:
This should be since `1.0.0`. We moved it from default constructor to auxiliary constructor. However, the signature did change.